cnames.ts

```diff
@@ -64,7 +64,7 @@ export default <CNAMEs> {
     target: "discordeno.github.io/guide",
   },
   "discorddn": {
```

Comment:
Should be moved to below `"discordbot"` and before `"discordeno"`

Reply:
Okay

Reply:
I should've noticed it, my bad 😅